akka-docs/src/main/paradox/serialization-jackson.md

```diff
@@ -68,16 +68,16 @@ For security reasons it is disallowed to bind the Jackson serializers to
 open ended types that might be a target for [serialization gadgets](https://medium.com/@cowtowncoder/on-jackson-cves-dont-panic-here-is-what-you-need-to-know-54cd0d6e8062),
 such as:
 
-* `java.lang.Object`
-* `java.io.Serializable`
-* `java.util.Comparable`.
+* @javadoc[java.lang.Object](java.lang.Object)
+* @javadoc[java.io.Serializable](java.io.Serializable)
+* @javadoc[java.lang.Comparable](java.lang.Comparable).
 
 The deny list of possible serialization gadget classes defined by Jackson databind are checked
 and disallowed for deserialization.
 
 @@@ warning
 
-Don't use `@JsonTypeInfo(use = Id.CLASS)` or `ObjectMapper.enableDefaultTyping` since that is a security risk
+Don't use [@JsonTypeInfo(use = Id.CLASS)](https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-annotations/latest/com/fasterxml/jackson/annotation/JsonTypeInfo.html) or [ObjectMapper.enableDefaultTyping](https://javadoc.io/doc/com.fasterxml.jackson.core/jackson-databind/latest/com/fasterxml/jackson/databind/ObjectMapper.html#enableDefaultTyping--) since that is a security risk
```

Comment:
Even these may use the `@javadoc` directive if Jackson gets added to the `paradoxProperties` in https://github.com/akka/akka/blob/main/project/Paradox.scala#L32

Reply:
@ennru Thank you for your advice.How can I do it? Something like that?:
`"javadoc.java.base_url" -> "https://docs.oracle.com/en/java/javase/11/docs/api/java.base/; SOME_OTHER_URL"`

Reply:
The latter, the bit between `javadoc.` and `.base_url` must be a prefix of the package. E.g. `com.fasterxml.jackson`
This is how it is configured in Alpakka: https://github.com/akka/alpakka/blob/master/build.sbt#L392-L393